snuba/query/allocation_policies/errors_allocation_policy.py

```diff
@@ -104,6 +114,8 @@ def _get_quota_allowance(self, tenant_ids: dict[str, str | int]) -> QuotaAllowan
                 return QuotaAllowance(
                     can_run=False, max_threads=0, explanation={"reason": why}
                 )
+        if tenant_ids["referrer"] in _PASS_THROUGH_REFERRERS:
+            return DEFAULT_PASSTHROUGH_POLICY.get_quota_allowance(tenant_ids)
```

Comment:
Shouldn't this early return check happen before tenant id validation? (Unless want to fail on bad tenant ids even if it's a passthrough referrer)

Reply:
The outcome is the same but knowing when we have bad tenant ids is good data. Also maybe there is no `referrer` either, don't want a KeyError happening